pkg/reconciler/v1alpha1/testing/functional.go

```diff
@@ -712,7 +712,7 @@ type PodOption func(*corev1.Pod)
 func WithFailingContainer(name string, exitCode int, message string) PodOption {
 	return func(pod *corev1.Pod) {
 		pod.Status.ContainerStatuses = []corev1.ContainerStatus{
-			corev1.ContainerStatus{
+			{
```

Comment:
nit: this can now be moved up to line 714 to collapse further.

Not sure how easy it is to update sockpuppet PRs, and looks like I can't do multi-line suggestions on the PR. So if it is difficult don't worry about it.